### PR TITLE
gen 2 johto gym work - needs more info

### PR DESCRIPTION
**Pull request by BaineGames**
added sprite assets for gen2 gyms
added 8 new johto badges to gameconstants
added 8 new gyms to gmy.ts
-still needs hp settings
-still needs on win text

### CODE REVIEW

**Comment by dennism1997** [comment]
The on win texts are the ones the gym leaders say when you defeat them. You can get them from the wiki. The hp settings are just randomly made up. The ones in kanto need balancing too

**Comment by Aegyo** [comment]
Can you add the Johto Elite 4 too?

File: src/scripts/gym/Gym.ts [hunk 1]
```diff
@@ -197,3 +197,155 @@ gymList["Elite Lance"] = new Gym(
...
+);
+
+//Johto Elite 4
+gymList["Elite Will"] = new Gym(
+    "Will",
+    "Elite Will",
+    [new GymPokemon("Xatu", 45330, 40),
+        new GymPokemon("Jynx", 48300, 41),
+        new GymPokemon("Exeggcutor", 52000, 41),
+        new GymPokemon("Slobro", 57000, 41),
```
Comment (Aegyo) [inline src/scripts/gym/Gym.ts]: Should be "Slowbro"

Comment (Aegyo) [reply src/scripts/gym/Gym.ts]: Also "Exeggcutor" -> should be "Exeggutor"

**Pull request closed without merging**